style-guides/scss.md

```diff
@@ -0,0 +1,123 @@
+<script>{
+	"title": "Chassis SCSS Style Guide"
+}</script>
```

Comment:
This does nothing unless we either move this to contribute.jquery.org or start using a jquery-wp-content website. Either way in this repo it should be removed.